common/primitives/core/src/omni_account.rs

```diff
@@ -57,3 +57,13 @@ impl OmniAccountConverter for DefaultOmniAccountConverter {
         identity.to_omni_account()
     }
 }
+
+// This type must be kept in sync with the `Permission` type used in the runtime.
+#[derive(Encode, Decode, Clone, Debug, PartialEq, Eq)]
+pub enum OmniAccountPermission {
+    All,
+    AccountManagement,
+    RequestNativeIntent,
+    RequestEthereumIntent,
+    RequestSolanaIntent,
```

Comment:
Yeah we could add any permissions we want, these are just examples. I assume this will be driven by the product, we can create an issue defining the permissions we are going to need